src/main/java/com/github/manolo8/darkbot/config/Config.java

```diff
@@ -99,6 +101,11 @@
             public @Option boolean ONLY_KILL_PREFERRED = false;
         }
     }
+    public @Option KeyBinds KEY_BINDS = new KeyBinds();
+    public static class KeyBinds{
+        public @Option Character CONFIGURATION_KEY = 'C';
+        public @Option Character JUMP_KEY = 'j';
```

Comment:
```suggestion
        public @Option Character JUMP_KEY = 'J';
```
If you don't capitalize this, the raw key click will have trouble